codegen/src/main/scala/overflowdb/codegen/Helpers.scala

```diff
@@ -131,7 +131,7 @@ object Helpers {
 
   def defaultValueImpl[A](default: Property.Default[A]): String =
     default.value match {
-      case str: String => s"\"$str\""
+      case str: String => s"$quotes$str$quotes"
```

Comment:
note: this may look odd,  but it's to make the codegen code compatible with scala 2.12 and 2.13 without deprecation warnings